docs/Schema-changes.md

```diff
@@ -0,0 +1,164 @@
+### Changes to Aries-vcr issuer schema
+
+The Aries-vcr application is undergoing changes to enhance user experience on applications such as Orgbook. The primary change to the Aries-vcr application is in the way that credential schemas are issued.  
```

Comment:
```suggestion
The Aries VCR application is undergoing changes to enhance user experience on applications such as Orgbook. The primary change to the Aries-vcr application is in the way that credential schemas are processed.  
```

Please replace 'Aries-vcr' with 'Aries VCR' throughout to keep things consistent with the rest of the documentation.